packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -273,7 +274,7 @@ export const ComponentShowcase = () => {
         </div>
         <div className="tooltip-row">
           <CWText>Solid background</CWText>
-          <CWTooltip
+          <CWTooltipOld
```

Comment:
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/435ec053-ce8b-4714-8d42-9b92a116b5d8)

Despite the fact you changed the component to old, the old tooltips looks like the new one. This is because you are breaking the encapsulation rule for styles, which I explain in the other comment

Reply:
Addressed